README.md

```diff
@@ -400,4 +401,4 @@ Become a bronze sponsor with a monthly donation of $10 and get your logo on our
 
 <img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/WeChatPay.JPG" width="250" align="middle"/>&nbsp;&nbsp;
 <img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/AliPay.JPG" width="250" align="middle"/>&nbsp;&nbsp;
-<a href="https://www.paypal.me/R136a1X" target="_blank"><img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/PayPal.JPG" width="250" align="middle"/></a>&nbsp;&nbsp;
\ No newline at end of file
+<a href="https://www.paypal.me/R136a1X" target="_blank"><img src="https://raw.githubusercontent.com/panjf2000/illustrations/master/payments/PayPal.JPG" width="250" align="middle"/></a>&nbsp;&nbsp;
```

Comment:
This is a little bit weird, additional lines are the same as deleted lines? How come?

Reply:
My editor (`VSCode`) will auto removes the additional `space` or `tab` at end of line after saving the file.

Reply:
I see.